common/client-libs/validator-client/src/nymd/traits/vesting_execute_client.rs

```diff
@@ -0,0 +1,268 @@
+pub use crate::nymd::cosmwasm_client::signing_client::SigningCosmWasmClient;
+use crate::nymd::cosmwasm_client::types::ExecuteResult;
+use crate::nymd::error::NymdError;
+use crate::nymd::fee_helpers::Operation;
+use crate::nymd::{cosmwasm_coin_to_cosmos_coin, NymdClient};
+use async_trait::async_trait;
+use cosmwasm_std::Coin;
+use mixnet_contract::{Gateway, IdentityKey, MixNode};
+use vesting_contract::messages::ExecuteMsg as VestingExecuteMsg;
+
+#[async_trait]
+pub trait VestingExecuteClient {
```

Comment:
I guess the question is, should this be `Execute` client or `Signing`. I'd prefer to keep `Signing` name for consistency sake, but that's subjective.